gpkit/interactive/printing.py

```diff
@@ -102,7 +102,7 @@ def _can_print_latex(o):
             return all(_can_print_latex(i) and _can_print_latex(o[i]) for i in o)
         elif isinstance(o, bool):
             return False
-        elif hasattr(o, '_latex'):
+        elif hasattr(o, 'latex'):
```

Comment:
This code was copied from SymPy, by the way. I think you're totally right though, `_latex` is gross!